src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md

```diff
@@ -17,9 +17,6 @@ PDF Services APIs supports the signed URLs of input and output files in the requ
 Accessing files from External Storage is currently <b>not</b> supported in the following APIs
 <ul>
     <li> <a href="../../../apis/#tag/PDF-Accessibility-Auto-Tag">PDF Accessibility Auto Tag</a> </li>
```

Comment:
Why this is not removed. we are supporting it in Auto Tag as well right ?
If yes, please remove this complete section.